### PR TITLE
Add validation support for schema files

### PR DESCRIPTION
**Pull request by mre**
This adds validation support for schema files.
It's a first step towards automated build support as discussed in #12.

### CODE REVIEW

File: schemas/events.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (dblock) [inline schemas/events.json]: I think most of our JSONs are 2 spaces, maybe auto-format these?

**Comment by dblock** [comment]
Looks good, the validation part could be a separate PR if you want, useful by itself.

**Comment by mre** [comment]
You're right. Narrowed down the scope to support validation only for now. I will create a separate PR for the travis stuff. If it looks good to you, I guess we could merge this already.

File: schemas/events.json (hunk 1 above)
Comment (mre) [inline schemas/events.json]: Yup. Changed the schema files from tabs to two spaces.

**Comment by dblock** [comment]
Lets enable the build with actual validation to finish this PR? Add .travis.yml with whatever that needs?

**Comment by mre** [comment]
Sure! Added the `.travis.yml` and the badge. Note that I already included initial support for running the cronjob. This will be used in the next pull request.

**Comment by dblock** [comment]
Merging.

**Comment by dblock** [comment]
Update https://github.com/slack-ruby/slack-api-ref/blob/master/CHANGELOG.md too when you wrap up cron.

**Pull request merged**